### PR TITLE
Milestone 2 completed

### PR DESCRIPTION
**Pull request by belsman**
## In this iteration, I implemented the visual look of App. Specifically, I
 - Implemented App, Display, ButtonPanel, Button stateless components
 - Utilized the prop-types on the components for props validation
 - Rendered the components to the browser

### CODE REVIEW

File: src/components/App.js
Comment (teekaytech) [inline src/components/App.js]: You should wrap all the child components in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @belsman :wave:

__You've made laudable progress in this milestone: everything seems to be working :+1:__

__However, a change is required before this project can be approved. Please check my inline comment below and fix it as appropriate.__



* * *
**Please make the required changes and submit another review request when you are done.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: src/components/App.js
Comment (belsman) [reply src/components/App.js:9]: But it did compile, that's why I didn't use it.

Comment (teekaytech) [reply src/components/App.js]: You're probably doing it the wrong way. Check my message on slack.

**Review by Torres-ssf — APPROVED** [review]
### Milestone Approved :+1: 
Hi @belsman :hand: 
Nice job on this milestone.
Good luck with the next one.

Happy coding :computer: 
**Sérgio Torres**

**Pull request merged**
